go.mod

```diff
@@ -5,13 +5,13 @@ go 1.17
 require (
 	github.com/Pallinder/go-randomdata v1.2.0
 	github.com/dgraph-io/ristretto v0.1.0
-	github.com/elastic/beats/v7 v7.0.0-alpha2.0.20211207150041-4641f759226c
+	github.com/elastic/beats/v7 v7.17.1
```

Comment:
Why do we rely on 7.17.1 as there is already a 7.17.2?

Reply:
My local copy is more recent, let me push the last changes.

Reply:
This is blocked by https://github.com/elastic/elastic-agent/pull/273, fleet server has dependencies on some of the packages from Elastic Agent. They used to be in the main beats repository.

```
 ❯ go mod tidy                                                                                                                                                          [15:17:22]
go: finding module for package github.com/elastic/beats/v7/x-pack/elastic-agent/pkg/core/server
go: finding module for package github.com/elastic/beats/v7/x-pack/elastic-agent/pkg/core/logger
go: finding module for package github.com/elastic/beats/v7/x-pack/elastic-agent/pkg/packer
```

Once everything is updated, I will make an issue to reduce the dependencies between the three repositories.